src/phoenix/experimental/callbacks/langchain_tracer.py

```diff
@@ -74,10 +82,52 @@ def _prompt_template(serialized: Dict[str, Any]) -> Iterator[Tuple[str, Any]]:
                 yield LLM_PROMPT_TEMPLATE_VARIABLES, kwargs.get("input_variables", [])
                 yield LLM_PROMPT_TEMPLATE_VERSION, "unknown"
                 break
-        except TypeError:
+        except (AttributeError, KeyError, TypeError):
             continue
 
 
+def _invocation_parameters(run_extra: Dict[str, Any]) -> Iterator[Tuple[str, str]]:
+    """Yields invocation parameters if present."""
+    yield LLM_INVOCATION_PARAMETERS, json.dumps(run_extra.get("invocation_params", {}))
+
+
+def _model_name(run_extra: Dict[str, Any]) -> Iterator[Tuple[str, str]]:
+    """Yields model name if present."""
+    for key in ["model_name", "model"]:
+        try:
+            yield LLM_MODEL_NAME, run_extra["invocation_params"][key]
+            break
+        except (KeyError, TypeError):
+            continue
```

Comment:
nit. same here, use a local variable for `invocation_params`
```suggestion
        if (name := invocation_params.get(key)):
            yield LLM_MODEL_NAME, name
            return
```